extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -254,6 +254,46 @@ public DateTime getStartTime(final String id)
     }
   }
 
+  public Map<String, Object> getMovingAverages(final String id)
```

Comment:
can you add nullable annotation

Reply:
I made this return an empty map instead for those cases